test/plugin/base_test.rb

```diff
@@ -223,10 +223,10 @@ def test_one_log_with_invalid_json_credentials
     setup_gce_metadata_stubs
     ENV['GOOGLE_APPLICATION_CREDENTIALS'] = INVALID_CREDENTIALS[:path]
     setup_logging_stubs do
-      d = create_driver
-      d.emit('message' => log_entry(0))
-      exception_count = 0
       begin
+        exception_count = 0
```

Comment:
Why did you need to move these in? Can `create_driver` or `emit` raise a `RuntimeError`? At least you should be able to move the initialization of `exception_count` back out...

Reply:
For grpc path, it used to raise a `RuntimeError('Unable to read the credential file')` at `d.run` with invalid json credentials. The new lib raises the same `RuntimeError` during `create_driver` instead.